### DISCUSSION TITLE
Bro when next release?

### DISCUSSION THREAD

**Discussion by fayndly** [material-components/material-web#5189] [Q&A] [answered]
I have a lot of errors in the library

**Comment by asyncliz** [comment, marked as answer]
I want to get automated nightly and scheduled monthly releases set up soon to help!

Next manual release we'll do at the end of November in a couple weeks.

What errors do you have? We'll want to make sure they're all getting fixed.

Edit: you can now `npm install @material/web@nightly` for the latest version each day.

**Reply by fayndly** [reply]
![Скриншот 14-11-2023 211159](https://github.com/material-components/material-web/assets/88275456/dd3306de-507d-4041-87f3-63948de98df0)
I just found one and it really bothers me

**Reply by fayndly** [reply]
code from docs

**Reply by asyncliz** [reply]
Can you post a reproduction? The code from the docs appears to be working: https://lit.dev/playground/#gist=c1e0856c46c87a202a383b8ad016bc66

**Reply by fayndly** [reply]
ok, it's all because of this style-
`* {
padding: 0px;
}`
thank you for opening my eyes

**Reply by asyncliz** [reply]
Nightly builds are now available! `npm install @material/web@nightly`
